blaze/expr/strings.py

```diff
@@ -83,6 +113,110 @@ class str_lower(UnaryStringFunction):
     def schema(self):
         return self._child.schema
 
+class str_isalnum(UnaryStringFunction): schema = bool_
+class str_isalpha(UnaryStringFunction): schema = bool_
+class str_isdecimal(UnaryStringFunction): schema = bool_
+class str_isdigit(UnaryStringFunction): schema = bool_
+class str_islower(UnaryStringFunction): schema = bool_
+class str_isnumeric(UnaryStringFunction): schema = bool_
+class str_isspace(UnaryStringFunction): schema = bool_
+class str_istitle(UnaryStringFunction): schema = bool_
+class str_isupper(UnaryStringFunction): schema = bool_
+
+class StrFind(ElemWise):
+    """
+    Find literal substring in string column.
+
+    """
+
+    _arguments = '_child', 'sub'
+    schema = datashape.Option(datashape.int64)
+
+
+@copydoc(StrFind)
+def str_find(col, sub):
+    if not isinstance(sub, basestring):
+        raise TypeError("'sub' argument must be a String")
+    return StrFind(col, sub)
+
+class Replace(ElemWise):
+    _arguments = '_child', 'old', 'new', 'max'
+
+    @property
+    def schema(self):
+        return self._child.schema
+
+def str_replace(col, old, new, max=None):
+    _validate(old, 'old', basestring, 'string')
```

Comment:
I just cloned existing logic (and didn't observe any failures when testing with either Python 2 or 3).